src/applications/vaos/services/appointment/transformers.js

```diff
@@ -396,17 +394,31 @@ function setContained(appt) {
     }
     case APPOINTMENT_TYPES.ccRequest: {
       return appt.ccAppointmentRequest.preferredProviders.map(provider => {
-        return {
+        let obj = {
```

Comment:
We could make this a more descriptive name, like `participant`.

Reply:
Done.